<launch>

  <node pkg="laserScan_Similarity" type="laserScan_Similarity_kitti" name="laserScan_Similarity_kitti" output="screen" >

    <param name="inputQueueSize" value="1"/>
    <param name="rangeThreshold" value="0.0"/>
    <param name="isVoxelFilter" value="false"/>
    <param name="voxelFilterConfig" value="$(find laserScan_Similarity)/cfg/voxel_filter.yaml" />

    <param name="sectionNumOfR" value="20"/>
    <param name="minRange" value="-15"/>
    <param name="maxRange" value="5.0" />

    <param name="sectionNumOfI" value="20"/>
    <param name="minIntensity" value="0.0"/>
    <param name="maxIntensity" value="1.0"/>

    <param name="baseDirKitti" value="/media/yh/yh/DataSet/kitti_benckmark_velodyne/dataset/sequences/00/velodyne/"/>

  </node>
</launch>
